anomalib/data/mvtec.py

```diff
@@ -380,7 +380,7 @@ def prepare_data(self) -> None:
             logging.info("Downloading the dataset.")
             with DownloadProgressBar(unit="B", unit_scale=True, miniters=1, desc="MVTec AD") as progress_bar:
                 urlretrieve(
-                    url=f"ftp://guest:GU.205dldo@ftp.softronics.ch/mvtec_anomaly_detection/{dataset_name}",
+                    url=f"https://www.mydrive.ch/shares/38536/3830184030e49fe74747669442f0f282/download/420938113-1629952094/{dataset_name}",
                     filename=self.root / dataset_name,
                     reporthook=progress_bar.update_to,
                 )
```

Comment:
```
            logging.info("Downloading the dataset.")
            url = "https://www.mydrive.ch/shares/38536/3830184030e49fe74747669442f0f282/download/420938113-1629952094"
            dataset_name = "mvtec_anomaly_detection.tar.xz"
            with DownloadProgressBar(unit="B", unit_scale=True, miniters=1, desc="MVTec AD") as progress_bar:
                urlretrieve(
                    url=f"{url}/{dataset_name}",
                    filename=self.root / dataset_name,
                    reporthook=progress_bar.update_to,
                )
```

You could do something like this

Reply:
Yes, I saw your  PR fixed the length of the url, graet job

Reply:
Should we close this one then?

Reply:
sure, no problem.